tests/SwaggerServiceTest.php

```diff
@@ -683,6 +685,61 @@ public function testAddDataPostRequestWithObjectParams()
         $service->addData($request, $response);
     }
 
+    public function testAddDataWithNotExistsMethodOnController()
```

Comment:
This test covers:
https://github.com/RonasIT/laravel-swagger/blob/688fe0b4146b8f9a71abb95e7971c47dcbbc1655/src/Traits/GetDependenciesTrait.php#L16-L18